packages/@lwc/engine-server/src/__tests__/fixtures/superclass/not-superclass/modules/x/mixinSuperclass/mixinSuperclass.js

```diff
@@ -0,0 +1,7 @@
+export const FancyMixin = Class => {
+    return class extends Class {
```

Comment:
TODO: What happens if this mixin has decorated props?

Reply:
I actually do that in production 😆